### PR TITLE
Improvements to Planet Protect, misc. bug fixes

### PR DESCRIPTION
**Pull request by medeor413**
- Planet Protect: Use UUIDs instead of aliases
- Planet Protect: Disallow vehicle spawning on protected planets
- Claims: Properly handle no arguments in /planet_access
- Claims: Overload Planet Protect commands
- Command Dispatcher: Allow command overloading using priority system
- General Commands: Rename /whoami to /serverwhoami
- General Commands: Remove /planet alias for /here
- Discord Bot: Fix commands not working in Discord
- Discord Bot: Send /me to Discord
- Discord Bot/IRC Bot: Echo commands used to both bots
- Core: Ensure SIGINT handler is set
- Data Parser: Parse entity message packets

NOTE: Claim owners will now manage those allowed to build on their planets using the /add_builder, /del_builder, and /list_builders commands. /add_helper, /rm_helper, /helper_list are no longer available.

### CODE REVIEW

**Comment by kharidiron** [comment]
@medeor413 I'm assuming you've tested all these things, yes? Lots of code to vet...

File: data_parser.py [hunk 1]
```diff
@@ -771,8 +771,11 @@ class EntityInteract(Struct):
 
 class EntityMessage(Struct):
     """packet type: 51"""
-    unknown = Byte
-    target_id = SBInt32
+    target_unique = Flag
```
Comment (kharidiron) [inline data_parser.py]: Ooo, this is a nifty little tidbit. How'd you find out about this flag?

**Comment by medeor413** [comment]
I test everything as I write it (although sometimes things slip through...)

File: data_parser.py (hunk 1 above)
Comment (medeor413) [inline data_parser.py]: Figured it out after messages from ancient gates were getting garbled. Turns out ancient gates have a unique entity ID.

**Pull request merged**